index.html

```diff
@@ -408,8 +408,9 @@ <h4>DataIntegrityProof</h4>
 `controller`.
           </p>
           <p>
-The `proofValue` property of the proof MUST be an ECDSA or deterministic ECDSA
-signature produced according to [[FIPS-186-5]] using the curves and hashes as
+The `proofValue` property of the proof MUST be an ECDSA signature, in general,
```

Comment:
```suggestion
In general, the `proofValue` property of the proof MUST be an ECDSA signature, 
```

Reply:
So it isn't that the `proofValue` property (well, really the *value* of this property) must, in general, be an ECDSA signature, but rather that it MUST be an ECDSA signature. The "in general" was meant to convey that it can be any of the two possible variants: deterministic or non-deterministic. Then the follow on "SHOULD" says the signature SHOULD be a deterministic one, but it doesn't *have* to be.

I agree the original wording didn't make that clear -- maybe we just drop the "in general" part and then say it SHOULD be the deterministic variant.

Reply:
That works for me, @TallTed what do you think?

Reply:
I'd be OK with wording like @dlongley suggests. Given that "specific variant" is a common synonym for "deterministic", I think we should restate this synonymity, because we can't be sure that the background of any given reader would let them understand either name without the other. I can't make a proper suggestion of my rewording because of the way GitHub handles (or fails to handle) deleted lines within sequence of lines chosen for suggestion... So I suggest that @Wind4Greg resolve the existing threads and adjust the PR directly, and then @dlongley and I can suggest tweaks (if any) on the new draft.